game/src/OptionMenu/VideoTab.gd

```diff
@@ -0,0 +1,6 @@
+extends HBoxContainer
+
+@export var initial_focus: Button
+
+func _ready():
+	initial_focus.grab_focus()
```

Comment:
This accomplishes nothing because `_ready` does not run on visibility, it runs on being loaded into the scene tree but since this can't be seen nothing can be focused.